docs/source/kedro_project_setup/configuration.md

```diff
@@ -487,15 +487,10 @@ where `config.yml` is formatted as below (for example):
 
 ```yaml
 run:
-  tag:
-    - tag1
-    - tag2
-    - tag3
+  tags: tag1, tag2, tag3
   pipeline: pipeline1
   parallel: true
-  node_names:
-    - node1
-    - node2
+  nodes_names: node1, node2
```

Comment:
Just a note: This will end up being changed back to `node_names` but better to have the change here as is, as the format of the command changes too.